### PR TITLE
splitPed-0.0.1.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/c4e6eb8f7196e7d858fd for a full test report.

File: easybuild/easyconfigs/s/splitPed/splitPed-0.0.1.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = '0.0.1'
+
+homepage = 'http://www.sph.umich.edu/csg/yli/splitPed/'
+description = """ splitPed splits a pedigree file into smaller files with subsets of 
+ markers. The current version is a pre-release."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.sph.umich.edu/csg/yli/splitPed/download/']
+sources = ['%s.V%s.tgz' % (name, version.replace(".",""))]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/splitPed/splitPed-0.0.1.eb]: space after comma above, PATH should point a list?, remove extra final line

File: easybuild/easyconfigs/s/splitPed/splitPed-0.0.1.eb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    'dirs': ["examples"]
+}
+
+# add the installation dir to PATH
+modextrapaths = { 
+    'PATH': "",
+}
+
+moduleclass = 'bio'
+
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/splitPed/splitPed-0.0.1.eb]: remote last empty line

lgtm ,  put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): no activity for > 6 months.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**